app/src/main/java/net/tensory/rxjavatalk/data/DebtFeed.java

```diff
@@ -18,10 +19,10 @@ public DebtFeed() {
         getTimedEmittedValue(3)
                 .subscribe(lannisterDebt::onNext);
 
-        getTimedEmittedValue(14)
```

Comment:
Flipped the values, since it seems like the Starks would ask for financial help before the Targaryn's totally random, I may be tired, lol!

Reply:
🤑